code/modules/surgery/organs/dwarf_organ.dm

```diff
@@ -0,0 +1,46 @@
+/obj/item/organ/alcoholvessel //essentially the opposite of the xeno's plasmavessel, but with alcohol
```

Comment:
At BARE MINIMUM, remove the changes to ethanol's processing, it shouldn't know about dwarf organs even remotely (and it looks like the organ already does what's happening there anyway...)